src/chrome/content/rules/AntiqueAppliances.com.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="AntiqueAppliances.com">
```

Comment:
Could you add a comment above this ruleset addressing any non functional hosts, for example, `autodiscover.antiqueappliances.com`?